server/execution/run.py

```diff
@@ -61,3 +47,12 @@ def remove_player(players):
             registered_players.pop(player.tan)
         except KeyError:
             logging.info(f"{player.tan} already removed")
+
+
+# TEST DATA
+if LOAD_TEST_DATA:
+    test_pending = dummy_entities.create_test_execution()
+    test_running = dummy_entities.create_test_execution(pending=False)
+
+    activate_execution(test_pending)
+    activate_execution(test_running)
```

Comment:
Does this work by just starting the server? I my understanding you have to execute this file individually to run this code, or am I missing something?

Reply:
By starting the server python runs every script code found in the files. Thats how the 'global' variables are initialized. The data i have refactored worked the same way. As prove:

![image](https://github.com/7erra/MANVSim/assets/99326069/b83dba1d-23f4-445e-a3c1-e63bc8b6d94f)